KUADRANT.md

```diff
@@ -242,15 +242,15 @@ The backend exports these permissions (defined in `plugins/kuadrant-backend/src/
 - `kuadrant.apiproduct.delete.all` - delete any APIProduct resource
 - `kuadrant.apiproduct.list` - list APIProduct resources (backend filters by ownership)
 
-**APIKeyRequest (access requests with ownership):**
-- `kuadrant.apikeyrequest.create` - create APIKeyRequest resources (resource permission scoped to APIProduct)
-- `kuadrant.apikeyrequest.read.own` - read own APIKeyRequest resources
-- `kuadrant.apikeyrequest.read.all` - read all APIKeyRequest resources
-- `kuadrant.apikeyrequest.update` - update any APIKeyRequest (approve/reject)
-- `kuadrant.apikeyrequest.update.own` - update own APIKeyRequest resources
-- `kuadrant.apikeyrequest.delete.own` - delete own APIKeyRequest resources
-- `kuadrant.apikeyrequest.delete.all` - delete any APIKeyRequest resource
-- `kuadrant.apikeyrequest.list` - list APIKeyRequest resources
+**APIKey (access requests with ownership):**
+- `kuadrant.apikey.create` - create APIKey resources (resource permission scoped to APIProduct)
```

Comment:
permissions for accessing pending for approval apikeys are still valid, aren't they?